examples/app-secaggplus/pyproject.toml

```
@@ -3,11 +3,11 @@ requires = ["poetry-core>=1.4.0"]
 build-backend = "poetry.core.masonry.api"
 
 [tool.poetry]
-name = "secaggplus-mt"
+name = "app-secaggplus"
```

about about changing the name to leave room for other SA protocols in the future?
```suggestion
name = "app-secagg"
```

---

That makes sense. I will change it to `app-secure-aggregation`. The name `secagg` is more or less specifically for the `SecAgg` protocol.

---

good idea